djmoney/contrib/exchange/backends/base.py

```diff
@@ -4,6 +4,8 @@
 from django.db.transaction import atomic
 from django.utils.http import urlencode
 
+import certifi
```

Comment:
```suggestion
try:
    import certifi
except ImportError:
    raise ImportError("Please install dependecy certifi - pip install certifi")
```

I would do this to remove the hard dependency and make it optional, would you agree?

Reply:
Yep, sure, I'll remove this dependency from `setup.py`

Reply:
Maybe it would make sense to have it as an extra dependency - like:

```
pip install djmoney[exchange_rates]
```

But I am not sure

Reply:
like to have both, the `try/except` and extras in setup.py

Reply:
That's a great idea! I like that! :)